modules/apps/headless/headless-builder/headless-builder-test/src/testFunctional/tests/[APIRest]APIGenerator/ExposeQueryParametersInAPIExplorerForGeneratedAPI.testcase

```diff
@@ -40,7 +40,7 @@ definition {
 	tearDown {
 		ApplicationAPI.deleteAllAPIApplication();
 
-		JSONObject.deleteAllCustomObjects();
```

Comment:
Why do we need to include the pageSize? Do we have more than 20 objects? :exploding_head: